templates/adp-backend-template-dotnet/skeleton/test/${{ values.dotnet_solution_name }}.Api.Tests/${{ values.dotnet_solution_name }}.Api.Tests.csproj

```diff
@@ -0,0 +1,40 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>net8.0</TargetFramework>
+		<ImplicitUsings>enable</ImplicitUsings>
+		<Nullable>enable</Nullable>
+
+		<IsPackable>false</IsPackable>
+		<IsTestProject>true</IsTestProject>
+		<!-- Exclude coverage from Core project. Outputs coverage only for Api projects -->
+		<Exclude>[${{ values.dotnet_solution_name }}.Core]*</Exclude>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<PackageReference Include="AutoFixture" Version="4.18.1" />
```

Comment:
Some people might prefer to use Faker.Net or Bogus. Same as with mapster, I dont think we should include this in the template, let users install whatever they prefer without having to untangle a dependency adds only for its demos

Reply:
I'm tempted to provide a best practice example mate. Some devs can be new/lower skilled, so may benefit.